docs/main_concepts.md

```diff
@@ -1,6 +1,15 @@
 # Main concepts
 
-Working with Streamlit is simple. First you sprinkle a few Streamlit commands into a normal Python script. Then you run the script: `streamlit run your_script.py [script args]`.
+Working with Streamlit is simple. First you sprinkle a few Streamlit commands into a normal Python script. Then you run the script:
+
+```
+$ streamlit run your_script.py [script args]
+```
+
+```eval_rst
+.. tip::
+   Did you know that you can pass either a local python script or a URL to `$ streamlit run` as an argument.
```

Comment:
Here too.

Reply:
same as above. Will be in the next push.